packages/next/taskfile.js

```diff
@@ -1,6 +1,4 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
-const notifier = require('node-notifier')
```

Comment:
This is still in the package.json but it's not used anywhere else, you can remove it there.

Reply:
Oh, I searched for it just in the `packages/next/package.json`. It's in the root `package.json` that makes sense.